osh/hub/scan/management/commands/find_failed_tasks.py

```diff
@@ -0,0 +1,45 @@
+from datetime import datetime
+
+import requests
+from django.core.management.base import BaseCommand
+from kobo.hub.models import Task
+
+start_date = datetime(2023, 10, 18)
+end_date = datetime(2023, 10, 19, 23, 59, 59)
+CSMOCK_VERSION = "csmock-3.5.0.20231018.081735.gf8bfc23.internal-1"
```

Comment:
Although this is a one time command, wouldn't it be better to not be tied to a specific version of csmock?